releases/about.md

```diff
@@ -0,0 +1,54 @@
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# About Releases
+
+## Release Cycle
+
+Splinter follows a pattern of alternating stable and development versions. Every
+even-numbered minor version (like 0.4 or 1.2) is a stable release, and every
```

Comment:
The versions like 1.2 and 2.3 here might be confusing, if the reader thinks its a reference to a real version. Suggest we use only 0.3, 0.4, 0.5 in the examples.

Reply:
Updated to only mention existing versions